algoliasearch/type_query.go

```diff
@@ -0,0 +1,60 @@
+package algoliasearch
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type MultipleQueriesRes struct {
+	Results []MultipleQueryRes `json:"results"`
```

Comment:
After today discussion, we agreed on only using this as an internal representation. The final result returned to the user should be the `[]MultipleQueryRes`, not the `MultipleQueriesRes`.